### PR TITLE
Feature/cargo odra for odra 0.8.0

### LINKED ISSUE #62: Adjust cargo-odra to Odra's `release/0.8.0`.

### PR DESCRIPTION
**Pull request by kubaplas**

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -28,6 +38,14 @@ test-testproject:
     cd testproject && cargo odra test -b casper
     cd testproject && cargo odra clean
 
+test-workspace-project:
+    cd testproject && rustup target add wasm32-unknown-unknown
+    cd testproject && rustup component add rustfmt --toolchain nightly-2023-03-01-x86_64-unknown-linux-gnu
```
Comment (kpob) [inline justfile]: Is this required to add the toolchain explicitly? Isn't it defined in the `rust-toolchain` file?

File: src/actions/update.rs [hunk 2]
```diff
@@ -2,34 +2,12 @@
 
 use std::path::PathBuf;
 
-use crate::{cli::UpdateCommand, command, log, paths};
+use crate::{cli::UpdateCommand, command, log};
 
 /// Runs `cargo update` on project and backends in .builder* folders.
```
Comment (kpob) [inline src/actions/update.rs]: The doc is outdated, there are no backends anymore. Very likely there are more places to edit.

File: src/odra_toml.rs [hunk 3]
```diff
@@ -4,17 +4,36 @@ use std::path::{Path, PathBuf};
...
-    /// Fully Qualified Name of the contract struct
     pub fqn: String,
 }
 
+impl Contract {
+    /// Extracts first part from fqn
+    pub fn module_name(&self) -> String {
+        self.fqn
+            .split_terminator("::")
+            .next()
```
Comment (kpob) [inline src/odra_toml.rs:23]: Does it work corectly, if the fqn has 3+ segments? For instance aa::bb::cc::Contract.

Comment (kubaplas) [reply src/odra_toml.rs:23]: This function should be named `crate_name`. Renamed it.

**Review by kpob — APPROVED** [review]

**Pull request merged**